nyc_taxi/anaconda-project.yml

```diff
@@ -42,7 +42,7 @@ commands:
     unix: panel serve --rest-session-info --session-history -1 dashboard.ipynb --show
     supports_http_options: true
   notebook:
-    notebook: .
+    notebook: index.ipynb
```

Comment:
I like that! In fact, I think that all the projects that an `index.ipynb` file should point to it.